README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+Semantic html is the framework of your website and should be organized and used in a fashion that makes it easy to follow.
```

Comment:
Your definition is right but dont over-complicate it. Semantic HTML just means giving meaning to the information with html tags so that search engines/browsers understand it.